rust/benches/benches/models/parse_models.rs

```diff
@@ -0,0 +1,44 @@
+#![allow(non_snake_case)]
+
+mod utils;
```

Comment:
it's idiomatic to not declare shared dependencies in every module of a directory, but instead declare them in a `mod.rs` and then `use` them in the modules of the directory. this has the benefit of a central place to look for shared dependencies. the directory structure could for example look like:
```
- models
    - mod.rs
    - parse_models.rs
    - serialize_models.rs
    - utils.rs
```
where the `mod.rs` then contains the following lines:
```rust
pub mod parse_models;
pub mod serialize_models;
mod utils;
```

Reply:
sounds good and makes sense!

Reply:
cleaned up in a more isolated way.